### ISSUE TITLE
Unable to Open Launcher when Video is Fullscreen on Second Monitor

### ISSUE THREAD

**Issue by p134c0d3** [pop-os/pop#3280]
**Distribution (run `cat /etc/os-release`):**
 NAME="Pop!_OS"
VERSION="22.04 LTS"
ID=pop
ID_LIKE="ubuntu debian"
PRETTY_NAME="Pop!_OS 22.04 LTS"
VERSION_ID="22.04"
HOME_URL="https://pop.system76.com"
SUPPORT_URL="https://support.system76.com"
BUG_REPORT_URL="https://github.com/pop-os/pop/issues"
PRIVACY_POLICY_URL="https://system76.com/privacy"
VERSION_CODENAME=jammy
UBUNTU_CODENAME=jammy
LOGO=distributor-logo-pop-os

**Issue/Bug Description:**
I'm experiencing an issue where when a video (tested with multiple sites) is put into fullscreen on my secondary monitor, when I hit the Super key, the launcher isn't activating on either monitor. This is when I have just the one application open (browser that has the fullscreen video). If I hope another application from the dock, I then am able to open the launcher again. This happens on both X11 and Wayland sessions. 

**Steps to reproduce (if you know):**
1. Open browser (Firefox, Brave, Floorp are the tested browsers)
2. Navigate to YouTube or Rumble
3. Start a video and make it fullscreen
4. Press Super key; launcher doesn't open

**Expected behavior:**
Whether video is fullscreen or not, pressing the Super key should open the launcher

**Other Notes:**

**Comment by devotoare** [comment]
I was having this issue with steam and came here to research and came across this. Found the answer here:

https://github.com/pop-os/pop/issues/2518

TLDR: Extensions > Pop Shell > Toggle "Allow launcher over fullscreen window"

**Comment by p134c0d3** [comment]
Awesome thank you so much!
